finagle-redis/src/main/scala/com/twitter/finagle/redis/Client.scala

```diff
@@ -120,6 +131,15 @@ class BaseClient(service: Service[Command, Reply]) {
     messages.grouped(2).toSeq flatMap { case Seq(a, b) => Some(a, b); case _ => None }
   }
 
+  /**
+   * Helper function to convert a Redis multi-bulk reply into a Map[String, String]
+   */
+  private[redis] def returnMap(messages: Seq[ChannelBuffer]) = {
```

Comment:
since this needs to be Strings eventually, can we take Seq[String] as an argument and use ReplyFormat.toString instead of ReplyFormat.toChannelBuffer?